### ISSUE TITLE
initSession() result

### ISSUE THREAD

**Issue by shchurov** [BranchMetrics/android-branch-deep-linking#276]
In `Branch` class there's method `public boolean initSession(BranchReferralInitListener callback)`. Javadoc says: `@return A {@link Boolean} value, indicating <i>false</i> if initialisation is unsuccessful.`
And here's the body of the method:
```
    public boolean initSession(BranchReferralInitListener callback) {
        initSession(callback, (Activity) null);
        return false;
    }
```
Obviously it always returns false, is it outdated documentation or a real issue?

**Comment by sojanpr** [comment]
@shchurov Thank you very much for mentioning this. This is definitely a code mistake. This return value indicate whether the init operation is in progress or not. The result for `initSession()` is provided in the `BranchReferralInitListener` callback. Will add this fix asap

**Comment by sojanpr** [comment]
@shchurov  We have opened a PR for fixing this issue( https://github.com/BranchMetrics/android-branch-deep-linking/pull/277). Will add this to the next release. Really appreciate your helps to improve the SDK.

**Comment by sojanpr** [comment]
@shchurov  We have released v2.0.0 with the  above fix. Thank you very much
